### DISCUSSION TITLE
Is there a plan to add Fido Metadata Service v3 support for verifying Authenticators?

### DISCUSSION THREAD

**Discussion by sheeri185adidam** [duo-labs/py_webauthn#232] [Q&A] [answered]
Hey guys,

Wanted to ask if there was any plans to add Fido Metadata Service v3 support for verifying authenticator attestation?

**Comment by MasterKale** [comment, marked as answer]
Not at this moment in time. I understand the value of it, but something like that is going to be highly RP-specific due to needing to persist MDS entries in a way that makes sense for the RP's architecture.

I suppose there's an opportunity here to add logic to handle the initial ingestion and subsequent refresh, as that's basically "download the blob and verify the signature." But it's not on any official roadmap ATM
